<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow3</class>
 <widget class="QMainWindow" name="MainWindow3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpireTool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/resources.qrc">
    <normaloff>:/logo/icon.ico</normaloff>:/logo/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>16</number>
    </property>
    <property name="topMargin">
     <number>16</number>
    </property>
    <property name="rightMargin">
     <number>16</number>
    </property>
    <property name="bottomMargin">
     <number>16</number>
    </property>
    <property name="spacing">
     <number>16</number>
    </property>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>20</number>
      </property>
      <item row="1" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="horizontalSpacing">
         <number>30</number>
        </property>
        <property name="verticalSpacing">
         <number>15</number>
        </property>
        <item row="3" column="0">
         <layout class="QGridLayout" name="layout_cards"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_potion_display">
          <property name="font">
           <font>
            <family>Kreon</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Potion Chance</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4" rowspan="4">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_card_display">
          <property name="font">
           <font>
            <family>Kreon</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Rare Chance (Next 3 Cards)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Kreon</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Event Chance (WIP, may be inaccurate)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="4">
         <layout class="QVBoxLayout" name="layout_options">
          <property name="spacing">
           <number>10</number>
          </property>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="layout_potions"/>
        </item>
        <item row="1" column="6" rowspan="3">
         <layout class="QVBoxLayout" name="layout_events"/>
        </item>
        <item row="0" column="1" rowspan="4">
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="layout_top">
      <property name="horizontalSpacing">
       <number>32</number>
      </property>
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="label_char">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Cantarell</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Unknown Character</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_boss_asc">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Cantarell</family>
          <pointsize>13</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Open the Options Window</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="label_char_image">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources/resources.qrc">:/icons/unknown.svg</pixmap>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>724</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionTutorial"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionStatistics"/>
    <addaction name="actionReference"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="actionTutorial">
   <property name="text">
    <string>Tutorial</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="text">
    <string>Statistics</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionReference">
   <property name="text">
    <string>Reference</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow3</receiver>
   <slot>showAboutWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow3</receiver>
   <slot>shutdown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTutorial</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow3</receiver>
   <slot>showTutorialWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReference</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow3</receiver>
   <slot>showReferenceWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStatistics</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow3</receiver>
   <slot>showStatsWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOptions</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow3</receiver>
   <slot>showOptionsWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showOptionsWindow()</slot>
  <slot>showReferenceWindow()</slot>
  <slot>showTutorialWindow()</slot>
  <slot>showStatsWindow()</slot>
  <slot>showAboutWindow()</slot>
  <slot>shutdown()</slot>
 </slots>
</ui>
